mysite/static/js/account/edit_photo.js

```diff
@@ -0,0 +1,64 @@
+var csrf_token = null;
+var width = null;
+
+function set_callback_for_image_upload(token){
+    $("#edit_photo").on("submit", upload_image);
+    csrf_token = token;
+    width = $("#progressBar").width();
+    $('#clear_message').click(hide_notification);
+}
+
+function hide_notification(event){
+    $(this).parent().parent().hide();
```

Comment:
.parent().parent() seems a little brittle? could you select a class or id here instead?